### PR TITLE
contributor elia added

### PR DESCRIPTION
**Pull request by elia314**

### CODE REVIEW

**Comment by elia314** [comment]
@mubaris  please review, thanks.

File: .all-contributorsrc [hunk 1]
```diff
@@ -104,6 +104,15 @@
...
         "review"
       ]
+    },
+    {
+      "login": "elia314",
+      "name": "Elia Ahadi",
+      "avatar_url": "https://avatars2.githubusercontent.com/u/9751505?v=3",
+      "profile": "https://github.com/elia314",
+      "contributions": [
+        "code"
```
Comment (mubaris) [inline .all-contributorsrc]: Change this to "doc"

File: README.md [hunk 2]
```diff
@@ -1,17 +1,31 @@
 # Curiosity
+<<<<<<< HEAD
```
Comment (mubaris) [inline README.md]: Remove this HEAD line

File: README.md [hunk 3]
```diff
@@ -1,17 +1,31 @@
...
+[![All Contributors](https://img.shields.io/badge/all_contributors-11-orange.svg?style=flat-square)](#contributors)
```
Comment (mubaris) [inline README.md]: Put all badges here

File: README.md [hunk 4]
```diff
@@ -1,17 +1,31 @@
...
+=======
+
+## Features
+Find amazing github projects as a feed using Google Chrome Extension. :zap: :zap:
+
+![gif](https://media.giphy.com/media/l2SpYDOZmp3H2cAAo/giphy.gif)
+
+## Updates/credits
 [![All Contributors](https://img.shields.io/badge/all_contributors-10-orange.svg?style=flat-square)](#contributors)
+>>>>>>> 488fa6142afc82696d26c38e3729142856c7a8e7
```
Comment (mubaris) [inline README.md]: Remove this line

File: README.md [hunk 5]
```diff
@@ -1,17 +1,31 @@
...
 
 [![Join the chat at https://gitter.im/curiositylab/curiosity](https://badges.gitter.im/curiositylab/curiosity.svg)](https://gitter.im/curiositylab/curiosity?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
```
Comment (mubaris) [inline README.md]: Put all badges up there

File: README.md [hunk 6]
```diff
@@ -1,17 +1,31 @@
...
 
 [![Codacy Badge](https://api.codacy.com/project/badge/Grade/a28af21730b647da8a84974696776cc0)](https://www.codacy.com/app/mubaris/curiosity?utm_source=github.com&utm_medium=referral&utm_content=mubaris/curiosity&utm_campaign=badger)
 [![Build Status](https://travis-ci.org/curiositylab/curiosity.svg?branch=master)](https://travis-ci.org/curiositylab/curiosity)
 [![Say Thanks](https://img.shields.io/badge/Say%20Thanks!-%F0%9F%A6%89-1EAEDB.svg)](https://saythanks.io/to/mubaris)
 
+<<<<<<< HEAD
```
Comment (mubaris) [inline README.md]: Remove

File: README.md [hunk 7]
```diff
@@ -1,17 +1,31 @@
...
 ## Features
 Find amazing github projects as a feed. :zap: :zap:
 
 ![gif](https://media.giphy.com/media/l2SpYDOZmp3H2cAAo/giphy.gif)
 
+=======
```
Comment (mubaris) [inline README.md]: Remove

File: README.md [hunk 8]
```diff
@@ -1,17 +1,31 @@
...
 
 [![Codacy Badge](https://api.codacy.com/project/badge/Grade/a28af21730b647da8a84974696776cc0)](https://www.codacy.com/app/mubaris/curiosity?utm_source=github.com&utm_medium=referral&utm_content=mubaris/curiosity&utm_campaign=badger)
 [![Build Status](https://travis-ci.org/curiositylab/curiosity.svg?branch=master)](https://travis-ci.org/curiositylab/curiosity)
 [![Say Thanks](https://img.shields.io/badge/Say%20Thanks!-%F0%9F%A6%89-1EAEDB.svg)](https://saythanks.io/to/mubaris)
 
+<<<<<<< HEAD
 ## Features
 Find amazing github projects as a feed. :zap: :zap:
 
 ![gif](https://media.giphy.com/media/l2SpYDOZmp3H2cAAo/giphy.gif)
```
Comment (mubaris) [inline README.md]: Avoid repetition of image

File: README.md [hunk 9]
```diff
@@ -35,6 +50,17 @@ Curiosity Chrome extension is available now. Which is made by [Aswanth](https://
 
 I've collected usernames of amazing stargazers :star: in Github. By using Github API, Curiosity will collect their stars and will show as a feed.
 
+<<<<<<< HEAD
```
Comment (mubaris) [inline README.md]: Remove this line

File: README.md [hunk 10]
```diff
@@ -35,6 +50,17 @@ Curiosity Chrome extension is available now. Which is made by [Aswanth](https://
...
+=======
+## What you need
+
+You need to submit Github Token to access Github API. Do this by generating a personal token listed in your own settings at: https://github.com/settings/tokens
+
+## How to use
+1) Add Chrome Extension
```
Comment (mubaris) [inline README.md]: It's not a chrome extension

File: README.md [hunk 11]
```diff
@@ -35,6 +50,17 @@ Curiosity Chrome extension is available now. Which is made by [Aswanth](https://
...
+2) Granted Curiosity your own github token
+3) Open up a new Chrome browser tab, you'll see a feed of Github projects which are being used heavily (starred). 
+>>>>>>> 488fa6142afc82696d26c38e3729142856c7a8e7
```
Comment (mubaris) [inline README.md]: Remove

**Comment by elia314** [comment]
made edits per your request @mubaris  (I think at least), please let me know if more edits are needed.

**Comment by mubaris** [comment]
Good work :+1:

**Pull request merged**